common/common.cpp

```diff
@@ -2089,6 +2098,12 @@ struct llama_model * llama_load_model_from_url(
         return NULL;
     }
 
+    if(params.hf_token!=NULL){
+        struct curl_slist *headers = NULL;
```

Comment:
Same it must be `curl_slist * headers`. Try to configure your IDE to follow exactly the rules of llama.cpp formating